FirstProject/assets/evaluation.less

```diff
@@ -34,11 +87,165 @@
 }
 
 a.border_bottom:hover {
-  border-bottom: 4px solid blue;
+  border-bottom: 4px solid @blue;
   border-bottom-width: 7px;
 }
 
 table {
   width: 100%;
 }
 
+/*Evaluation html*/
+
+.add_button {
```

Comment:
Yes. If you noticed I started to change a lot of class names.
I will remove the snake_case.